Packages/MIES/MIES_Async.ipf

```diff
@@ -37,20 +37,23 @@ static StrConstant ASYNC_ERRORMSG_STR = "errmsg"
 /// @param numThreads number of threads to setup for processing data, must be >= 1 and <= ASYNC_MAX_THREADS
 ///
 /// @param disableTask [optional, default = 0] when set to 1 the background task processing readouts is not started
+///
+/// @return 1 if ASYNC framework was started, 0 if ASYNC framework was already running, in this case the number of threads is not changed
 Function ASYNC_Start(numThreads, [disableTask])
 	variable numThreads, disableTask
 
 	variable i
 
-	NVAR tgID = $GetThreadGroupID()
-	if(!isNaN(tgID))
-		// framework already running
+	DFREF dfr = GetAsyncHomeDF()
+	if(IsASYNCRunning())
 		return 0
 	endif
 
 	ASSERT(numThreads >= 1 && numThreads <= ASYNC_MAX_THREADS, "numThread must be > 0 and <= " + num2str(ASYNC_MAX_THREADS))
 	disableTask = ParamIsDefault(disableTask) ? 0 : !!disableTask
 
+
```

Comment:
unexpected newline

Reply:
I was still in the progress of resorting/cleaning up the commits.